src/components/Courses/Courses.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
```

Comment:
Strongly advise you to use CSS modules: https://create-react-app.dev/docs/adding-a-css-modules-stylesheet/. This is the best approach, in my opinion, to declare style per each component